src/Microsoft.Sbom.Targets/SbomCLIToolTask.cs

```diff
@@ -29,7 +29,41 @@ protected override string GenerateFullPathToTool()
     /// <returns>string list of args</returns>
     protected override string GenerateCommandLineCommands()
     {
-        return "Command";
+        var arguments =
+                "generate " +
+                $"-BuildDropPath {this.BuildDropPath} " +
+                $"-BuildComponentPath {this.BuildComponentPath} " +
+                $"-PackageName {this.PackageName} " +
+                $"-PackageVersion {this.PackageVersion} " +
+                $"-PackageSupplier {this.PackageSupplier} " +
+                $"-NamespaceUriBase {this.NamespaceBaseUri} " +
+                $"-DeleteManifestDirIfPresent {this.DeleteManifestDirIfPresent} " +
+                $"-FetchLicenseInformation {this.FetchLicenseInformation} " +
+                $"-EnablePackageMetadataParsing {this.EnablePackageMetadataParsing} " +
+                $"-Verbosity {this.Verbosity} ";
+
+        // For optional arguments, append them only if they are specified by the user
+        if (!string.IsNullOrEmpty(this.ManifestDirPath))
+        {
+            arguments += $"-ManifestDirPath {this.ManifestDirPath} ";
+        }
+
+        if (!string.IsNullOrEmpty(this.ExternalDocumentListFile))
+        {
+            arguments += $"-ExternalDocumentListFile {this.ExternalDocumentListFile} ";
+        }
+
+        if (!string.IsNullOrEmpty(this.NamespaceUriUniquePart))
+        {
+            arguments += $"-NamespaceUriUniquePart {this.NamespaceUriUniquePart} ";
+        }
+
+        if (!string.IsNullOrEmpty(this.ManifestInfo))
+        {
+            arguments += $"-ManifestInfo {this.ManifestInfo} ";
+        }
```

Comment:
Consider using StringBuilder rather than concatenating strings: https://learn.microsoft.com/en-us/dotnet/standard/base-types/stringbuilder

Reply:
Good idea! I saw there was a [CommandLineBuilder](https://learn.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.commandlinebuilder?view=msbuild-17-netcore) class we can use to construct command line arguments. I saw this was used in the SDK project example as well